google/cloud/aiplatform/training_jobs.py

```diff
@@ -1429,9 +1438,40 @@ def _prepare_training_task_inputs_and_output_dir(
             training_task_inputs["network"] = network
         if tensorboard:
             training_task_inputs["tensorboard"] = tensorboard
+        if enable_web_access:
+            training_task_inputs["enable_web_access"] = enable_web_access
 
         return training_task_inputs, base_output_dir
 
+    @property
+    def web_access_uris(self) -> Optional[Dict[str, str]]:
+        """Get the web access uris of the backing custom job.
+
+        Returns:
+            (Dict[str, str]):
+                Web access uris of the backing custom job.
+        """
+        if self._custom_job:
```

Comment:
Since this property isn't used by the logging logic, it most likely needs to instantiate the custom job instead of reference `self._custom_job` as it might be used after instantiating an already running training job.

ie
```Python
my_job = aiplatform.CustomTrainingJob.get(...)
my_job.state # RUNNING
my_job.web_access_uris
```